TL improvement (#538)
07th-mod/higurashi-console-arcs: Update/omot_009_togoodend.txt

			NULL, "\"Yes, well...", Line_WaitForInput);
	ModPlayVoiceLS(4, 15, "ps3/s14/15/2115000062", 256, TRUE);
	OutputLine(NULL, "そりゃ数十分もあれば…」",

> 			NULL, " that would likely take just several tens of minutes...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " that would likely take just half an hour or so...\"", Line_Normal);
```
